crates/sui-core/src/consensus_adapter.rs

```diff
@@ -471,6 +466,46 @@ impl ConsensusAdapter {
         )
     }
 
+    // According to the throughput profile we want to either allow some transaction duplication or not)
+    // When throughput profile is Low and the validator is in position = 1, then it will submit to consensus with much lower latency.
+    // When throughput profile is High then we go back to default operation and no-one co-submits.
+    fn override_by_throughput_profiler(&self, position: usize, latency: Duration) -> Duration {
+        const LOW_THROUGHPUT_LATENCY_MS: u64 = 800;
```

Comment:
The initial idea is for this to be `0` - so the validator at position 1 is co-submitting with validator at position 0. I want to load test this before taking a decision